app/services/load_events_from_github_service.rb

```diff
@@ -39,7 +38,28 @@ def add_new_event?(event)
     event.event == LABELED_EVENT && Event.find_by(github_id: event.id).nil?
   end
 
+  def all_issues_for_needed_period
+    all_owners_and_repos_array.reduce([]) do |array, owner_repo_array|
+      array << github.issues.list(
+        user: owner_repo_array.first,
+        repo: owner_repo_array.last,
+        since: since_issues_value,
+        per_page: EVENTS_PER_PAGE
+      ).reduce([]) do |arr, issue|
+        arr << { user: owner_repo_array.first, repo: owner_repo_array.last, number: issue.number }
+      end
+    end.flatten.compact
+  end
+
+  def since_issues_value
+    Event.any? ? last_event_formatted_created_date : DEFAULT_SINCE_VALUE
+  end
+
+  def last_event_formatted_created_date
+    (Date.today - Event.order(:created).last.created).to_i.days.ago.iso8601
```

Comment:
Unfortunately, no. `2015-08-09T00:00:00+03:00` vs `2015-08-09T13:22:47Z`
We need to match [this regExp](https://github.com/piotrmurach/github/blob/master/lib/github_api/client/issues.rb#L36)